### ISSUE TITLE
Rare decoding errors with `get_match`?

### ISSUE THREAD

**Issue by deklanw** [MingweiSamuel/Riven#11]
`RU 184554369`

```
 RiotApiError {
    reqwest_error: reqwest::Error {
        kind: Decode,
        source: Error("Matching variant not found", line: 1, column: 1225),
    },
    retries: 0,
    response: None,
    status_code: Some(
        200,
    ),
}
```

`OC1 227513707`

```
 RiotApiError {
    reqwest_error: reqwest::Error {
        kind: Decode,
        source: Error("Matching variant not found", line: 1, column: 9112),
    },
    retries: 0,
    response: None,
    status_code: Some(
        200,
    ),
}
```

`JP1 188794778`

```
 RiotApiError {
    reqwest_error: reqwest::Error {
        kind: Decode,
        source: Error("Matching variant not found", line: 1, column: 1225),
    },
    retries: 0,
    response: None,
    status_code: Some(
        200,
    ),
}
```

**Comment by MingweiSamuel** [comment]
Looks like a juicy one. I'm busy today but I can fix it tomorrow. Missing enum variant, might be missing on the official docs :D

**Comment by MingweiSamuel** [comment]
Released 1.4.0 https://crates.io/crates/riven/1.4.0

Wasn't too juicy. [Tier](https://docs.rs/riven/1.4.0/riven/consts/enum.Tier.html) enum was missing an "UNRANKED" case, docs were fine
